api/publish.go

```diff
@@ -339,10 +339,14 @@ func apiPublishUpdateSwitch(c *gin.Context) {
 		published.AcquireByHash = *b.AcquireByHash
 	}
 
+	if b.MultiDist != nil {
+		published.MultiDist = *b.MultiDist
+	}
```

Comment:
Should we not drop support for setting `MultiDist` after published repository creation at all?

Reply:
I thought about it, but for existing publish points this will automatically also persist the setting, so we can depreciate it later (or keep it changeable)...

Reply:
Ah, ok. Let's do the persistence first and discuss immutability later 😃.